server/app/featureflags/FeatureFlag.java

```diff
@@ -31,10 +31,7 @@ public enum FeatureFlag {
   NONGATED_ELIGIBILITY_ENABLED,
 
   // Phone number question type.
-  PHONE_QUESTION_TYPE_ENABLED,
-
-  // New login form
-  NEW_LOGIN_FORM_ENABLED;
```

Comment:
The "new login form" never came to fruition because we are getting rid of the form altogether; removing the feature flag here.